examples/list.rs

```diff
@@ -18,8 +18,6 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 // SOFTWARE.
 
-extern crate globwalk;
```

Comment:
in modern Rust, `extern crate` is no longer necessary (also in some other places)